src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -897,3 +904,34 @@ def dump_flow_result(flow_folder, prefix, flow_result=None, node_result=None):
     if output:
         with open(dump_folder / f"{prefix}.output.json", "w") as f:
             json.dump(output, f, indent=2)
+
+
+def _resolve_base64_image_in_test_result(test_result, flow_folder, sub_dir):
+    """
+    Resolve base64 to image path in the test result.
```

Comment:
_resolve sounds like reading from file to memory obj. But you are doing the opposite here. Maybe name this as: 

_save_base64_image_to_file()